### ISSUE TITLE
Very generic error "No such file or directory"

### ISSUE THREAD

**Issue by meh** [byronwasti/movine#19]
```
[devshell] query/service $ movine status
Error: IO Error: No such file or directory (os error 2)
```

What is this trying to tell me and how can I debug it? :panda_face: 

I tried adding `RUST_LOG=debug` but that didn't help.

**Comment by meh** [comment]
Oh, got it, it's what happens when a file is missing in the migration directory.

**Comment by byronwasti** [comment]
Yeah the default IO error isn't particularly descriptive. I will look into making the error handling here return something nicer to work with. Thanks for the bug!
